Floofbot/Modules/Utilities.cs

```diff
@@ -256,4 +258,163 @@ public async Task ServerList()
             }, true);
         }
     }
+    [Summary("Conversion commands")]
+    [Discord.Commands.Name("Conversions")]
+    public class Conversions : InteractiveBase
+    {
+        private static readonly Discord.Color EMBED_COLOR = Color.Magenta;
```

Comment:
Not needed, there's already a readonly static at the top

Reply:
Actually it is, because it's a class. However, related to the above comment, I'll make it a command to it won't be needed anymore.